code/_onclick/hud/radial.dm

```diff
@@ -279,7 +276,7 @@ GLOBAL_LIST_EMPTY(radial_menus)
 		return
 	current_user = M.client
 	//Blank
-	menu_holder = image(icon='icons/effects/effects.dmi',loc=anchor,icon_state="nothing",layer = ABOVE_HUD_LAYER)
```

Comment:
Stuff that set that layer didn't really care about value but about having same arbitrary static layer for everything since appearances of contents can have literally anything in there. Removing it can cause issues.